platform/darwin/docs/guides/Migrating to Expressions.md.ejs

```diff
@@ -0,0 +1,113 @@
+<%
+  const os = locals.os;
+  const iOS = os === 'iOS';
+  const macOS = os === 'macOS';
+  const cocoaPrefix = iOS ? 'UI' : 'NS';
+  const guide = 'MigratingToExpressions';
+-%>
+<!--
+  This file is generated.
+  Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
+-->
+
+# Migrating from Style Functions to Expressions
+
+[Runtime Styling](runtime-styling.html) enables you to modify every aspect of the map’s appearance dynamically as a user interacts with your application. Users can specify in advance how a layout or paint attribute will vary as the zoom level changes or how the appearance of individual features vary based on metadata provided by a content source. With Maps SDK of iOS v4.0.0, style functions have been replaced with expressions. These provide even more tools for users who want to style their maps dynamically. This guide outlines some tips for migrating from style functions to expressions, and offers an overview of some things that users can do with expressions.
```

Comment:
> offers an overview of some things that users can do with expressions

If a “user” in this case is the developer, let’s say “developer” (to distinguish between _our_ users and a developer’s users).